Can't build docs inside make virtualenv

Looks like something broken just now
```
$ history | tail
1229  2017.10.03 16:28:16  git clone https://github.com/buildbot/buildbot.git
 1230  2017.10.03 16:28:32  cd buildbot/
 1231  2017.10.03 16:28:33  ll
 1232  2017.10.03 16:28:39  make virtualenv 
 1233  2017.10.03 16:29:21  source .venv/bin/activate

$ make docs
make -C master/docs
make[1]: Entering directory `/tmp/buildbot/master/docs'
rm -rf _build/*
if command -v towncrier >/dev/null 2>&1 ;\
then \
cd ../../; towncrier --draft |grep  'No significant changes.' || yes n | towncrier ;\
fi
Loading template...
Finding news fragments...
Rendering news fragments...
Traceback (most recent call last):
  File "/tmp/buildbot/.venv/bin/towncrier", line 11, in <module>
    load_entry_point('towncrier==16.12.0', 'console_scripts', 'towncrier')()
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/towncrier/__init__.py", line 36, in _main
    return __main(draft, directory, project_version, project_date)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/towncrier/__init__.py", line 73, in __main
    fragments = split_fragments(fragments, definitions)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/towncrier/_builder.py", line 74, in split_fragments
    ticket, category = parts
ValueError: too many values to unpack
Loading template...
Finding news fragments...
Rendering news fragments...
Traceback (most recent call last):
  File "/tmp/buildbot/.venv/bin/towncrier", line 11, in <module>
    load_entry_point('towncrier==16.12.0', 'console_scripts', 'towncrier')()
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/towncrier/__init__.py", line 36, in _main
    return __main(draft, directory, project_version, project_date)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/towncrier/__init__.py", line 73, in __main
    fragments = split_fragments(fragments, definitions)
  File "/tmp/buildbot/.venv/lib/python2.7/site-packages/towncrier/_builder.py", line 74, in split_fragments
    ticket, category = parts
ValueError: too many values to unpack
make[1]: *** [towncrier] Error 1
make[1]: Leaving directory `/tmp/buildbot/master/docs'
make: *** [docs] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -82,7 +82,7 @@ virtualenv: $(VENV_NAME)   # usage: make virtualenv VENV_PY_VERSION=python3.4
 		-e 'master[tls,test,docs]' \
 		-e 'worker[test]' \
 		buildbot_www packaging \
-		'git+https://github.com/tardyp/towncrier'
+		'towncrier>=17.8.0'
 
 release_notes: $(VENV_NAME)
 	test ! -z "$(VERSION)"  #  usage: make release_notes VERSION=0.9.2

